scripts/submit_sim_imcalc.py

```diff
@@ -109,9 +109,6 @@ def submit_im_calc_slurm(
             const.SlBodyOptConsts.advanced_IM.value
         ] = f"-a {body_options['models']} --OpenSees {qconfig['OpenSees']} "
 
-        print(
-            f"test {params[const.SlBodyOptConsts.advanced_IM.value]['match_obs_stations']}"
-        )
```

Comment:
logger.debug should replace the prints, rather than removing them. Perhaps a bit more context around the strings being logged too.
Same below